impeller/toolkit/egl/display.cc

```diff
@@ -141,9 +141,9 @@ std::unique_ptr<Config> Display::ChooseConfig(ConfigDescriptor config) const {
     if (sample_count > 1) {
       attributes.push_back(EGL_SAMPLE_BUFFERS);
       attributes.push_back(1);
+      attributes.push_back(EGL_SAMPLES);
```

Comment:
We want to specify sample count to 1 explicitly right? If the caller wants 1 and we specify no opinion, the implementation is allowed to select anything it wants (I think). We will not be respecting the caller opinion for 1.

Reply:
You can't, egl fails on the emulator if you specify 1.  You have to leave it undefined.  I expect undefined would not give you MSAA.  It isn't officially documented as such though.

Reply:
Do you want me to add an undefined option in the `ConfigDescriptor` to be explicit?

Reply:
> egl fails on the emulator if you specify 1

Didn't know that.

> Do you want me to add an undefined option in the ConfigDescriptor to be explicit?

No. This is fine.

LGTM still.